src/irrlicht_changes/CGUITTFont.cpp

```diff
@@ -31,7 +31,11 @@
    john@suckerfreegames.com
 */
 
+#include <cstdlib>
+#include <freetype/freetype.h>
```

Comment:
Could this line of include be removed? Header of freetype is already introduced by `CGUITTFont.h`

Reply:
Stuff like this is why "Clean up some stuff" is still on the todos :P

Thanks for taking a peek.